### PR TITLE
feat: upgrade module, add test instructions

### PR DESCRIPTION
**Pull request by laszlokr**
@janikvonrotz this module also does not carry the sale partner address info from the sales order to the invoice, but only let's to choose the sale partner address in the invoice. Since this field does is not shown on the sales order, it looks like this is taken care of by another module, otherwise the view needs to be extended

### CODE REVIEW

File: sale_invoice_prepare_partner/tests/TEST_INSTRUCTIONS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- Open Sales > Orders > Quotations, create a new quotation for Azure Interior
```
Comment (janikvonrotz) [inline sale_invoice_prepare_partner/tests/TEST_INSTRUCTIONS.rst]: There should be a step where the `partner_invoice_id` is set on the sale.order.

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

**Comment by janikvonrotz** [comment]
The other module is `account_move_invoice_partner`.

File: sale_invoice_prepare_partner/tests/TEST_INSTRUCTIONS.rst
Comment (janikvonrotz) [inline sale_invoice_prepare_partner/tests/TEST_INSTRUCTIONS.rst:1]: I am not sure if this module is actually needed. Does this test work without the module?

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

File: sale_invoice_prepare_partner/tests/TEST_INSTRUCTIONS.rst
Comment (laszlokr) [reply sale_invoice_prepare_partner/tests/TEST_INSTRUCTIONS.rst:1]: @janikvonrotz it seems the module is still needed, and the sale.order view has to provide the `partner_invoice_id` field to make it work.

**Review by janikvonrotz — APPROVED** [review]

**Pull request merged**